tokio/src/runtime/scheduler/multi_thread/queue.rs

```diff
@@ -106,9 +106,12 @@ pub(crate) fn local<T: 'static>() -> (Steal<T>, Local<T>) {
 }
 
 impl<T> Local<T> {
-    /// Returns true if the queue has entries that can be stolen.
-    pub(crate) fn is_stealable(&self) -> bool {
-        !self.inner.is_empty()
+    /// Returns the number of entries in the queue
+    pub(crate) fn tasks_num(&self) -> (usize /* stealable */, usize /* total */) {
```

Comment:
It should be fine to return a single `usize` here. I am not sure which comment you are referencing.

Reply:
> "protect" some tasks from stealing
https://github.com/tokio-rs/tokio/blob/fb4d43017d6cb7b7cbdfbe5fc189bca0291ff215/tokio/src/runtime/scheduler/multi_thread/queue.rs#L122-L128